File: GildedRoseKata/GildedRose.VB.WebUI/Views/Home/Item.vb
Public Class Item
    Private _name As String
    Public Property Name() As String
        Get
            Return _name
        End Get
        Set(ByVal value As String)
            _name = value
        End Set
    End Property


    Private _sellin As Integer
    Public Property Sellin() As Integer
        Get
            Return _sellin
        End Get
        Set(ByVal value As Integer)
            _sellin = value
        End Set
    End Property

    Private _quality As Integer
    Public Property Quality() As Integer
        Get
            Return _quality
        End Get
        Set(ByVal value As Integer)
            _quality = value
        End Set
    End Property

    Public Overrides Function ToString() As String
        Return _name & "#" & _sellin & "#" & _quality
    End Function
End Class

File: GildedRoseKata/GildedRose.CSharp.WebUI/Views/Home/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace GildedRose.CSharp.WebUI.Views.Home
{
    public class Item
    {
        public string Name { get; set; }

        public int SellIn { get; set; }

        public int Quality { get; set; }

        public override String ToString()
        {
            return Name + "#" + SellIn + "#" + Quality;
        }
    }
}